### PR TITLE
@xlc 's linked map metadata inclusion

### PR DESCRIPTION
**Pull request by gavofyork**
Reopened since it needed runtime bump/rebuild.

CC @xlc

### CODE REVIEW

**Comment by xlc** [comment]
I can do a rebase & version bump on my branch if you want

File: srml/metadata/src/lib.rs [hunk 1]
```diff
@@ -325,7 +328,7 @@ impl Decode for RuntimeMetadataDeprecated {
 /// The metadata of a runtime version 1.
 #[derive(Eq, Encode, PartialEq)]
 #[cfg_attr(feature = "std", derive(Decode, Debug, Serialize))]
```
Comment (bkchr) [inline srml/metadata/src/lib.rs]: The documentation for this type is not correct and needs to be updated.

**Comment by xlc** [comment]
I have rebased and bumped version on my branch https://github.com/paritytech/substrate/pull/2037

File: srml/metadata/src/lib.rs (hunk 1 above)
Comment (xlc) [reply srml/metadata/src/lib.rs]: Fixed on my PR

**Pull request closed without merging**